modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/MultiSelectTag.java

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib;
+
+import com.liferay.frontend.taglib.clay.internal.servlet.taglib.BaseContainerTag;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.JspWriter;
+
+/**
+ * @author Kresimir Coko
+ */
+public class MultiSelectTag extends BaseContainerTag {
+
+	@Override
+	public int doStartTag() throws JspException {
+		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
+
+		return super.doStartTag();
+	}
+
+	public String getClearAllTitle() {
+		return _clearAllTitle;
+	}
+
+	public String getCloseButtonAriaLabel() {
+		return _closeButtonAriaLabel;
+	}
+
+	public boolean getDisabledClearAll() {
+		return _disabledClearAll;
+	}
+
+	public Object getFilter() {
```

Comment:
Clay MultiSelect `filter` param expects a function. We can't pass js functions to a Java Tag so we don't add this kind of parameters to the tag API.

If someone wants to pass, for example, a filter, it should be done through a `propsTransformer`

Reply:
Did u end up making that document about these new tags, remember, the one we talked about a few weeks ago?

Reply:
@carloslancha, So the `propsTransformer` is there for users of the component to be able to supply a set of props they want to provide the component **without us worrying about them**?

All of the Multi Select's props are **not** required, does that mean I need to add a sample of **all** of them, or none of them?

Reply:
No, `propsTransformer` is a mechanism for the developers to be able to add / modify / remove props, add logic, whatever to the component. Since this is used via java tags this is the way to wire javascript code to it.

You don't need to add samples for all of them, I just recommend you to add samples to TEST that everything is wired and working as expected.